_build/stac.hbs

```diff
@@ -38,5 +42,18 @@
       "rel": "license"
     }
   ],
-  "cube:dimensions": {{{copyJson CubeDimensions}}}
+  "cube:dimensions": {{{copyJson CubeDimensions}}},
+  {{#if sci:citation }}
+  "sci:citation": {{{sci:citation}}},
+  {{/if}}
+  {{#if Summaries }}
+  "summaries": {{{copyJson Summaries}}},
+  {{/if}}
+  "assets": {
+   {{#if Image }}

```

Comment:
The 'image" parameter implies it's a big image and not a small thumbnail. Is the "thumbnail" field in JSON meant to represent a real small thumbnail, or this is just a general field for an image representation of the collection?

However, if a small thumbnail image is to be generated from an optional bigger image, this is probably possible to do.

Reply:
I checked a few random images and they were all around 300 KB or smaller so I think at least for now, it would be okay to use them here as thumbnails (the images are already used by the openEOP as such and so far there were no problems with it).